shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/adapter/AdapterContainerFactory.java

```diff
@@ -40,15 +40,15 @@ public final class AdapterContainerFactory {
      * @param databaseType database type
      * @param storageContainer storage container
```

Comment:
Why use storage container and database type together, is storage container include database type?

Reply:
for the most parameters in AdapterContainerFactory will be refactored in next PR.